sourcegraph/learn
# How to troubleshoot Java StackOverflowError

## Type of change

Add a tutorial
Resolves #253 
### What should this PR do?

Update the authors.yaml file
Add a new tutorial which details how to address the  How to troubleshoot Java StackOverflowError #253 

### What are the acceptance criteria? 


This PR adds a new tutorial that can be viewed on the site

### How should this PR be tested?

View the tutorial locally to check that it renders correctly
Check that the writing is grammatically correct and makes sense to the reader
## Pull request process

**Reviewers**:

1. Test functionality using the criteria above.
2. Offer tips for efficiency, feedback on best practices, and possible alternative approaches and things that may not have been considered.
3. For shorter, "quick" PRs, use your best judgement on #​2.
4. Use a collaborative approach and provide resources and/or context where appropriate.
5. Provide screenshots/grabs where appropriate to show findings during review.

**Reviewees**:

1. Prefer incremental and appropriately-scoped changes.
2. Leave a comment on things you want explicit feedback on.
3. Respond clearly to comments and questions.
